ldi-orchestrator/ldio-connectors/ldio-kafka/src/test/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/KafkaInIntegrationTestSteps.java

```diff
@@ -133,11 +133,11 @@ public void theComponentExecutorWillHaveBeenCalled(int i) {
 	}
 	@When("I pause the pipeline")
 	public void pauseInput() {
-		kafkaIn.updateStatus(PipelineStatus.HALTED);
+		kafkaIn.updateStatus(PipelineStatusTrigger.HALT);
 	}
 	@When("I unpause the pipeline")
```

Comment:
Could this be formatted please, as I see there is no whitespace between these methods